docs/en/02_Developer_Guides/14_Files/01_File_Management.md

```diff
@@ -97,69 +105,99 @@ above methods (after content and filename). The available constants are:
-| `File`         | The `[DBFile](api:SilverStripe\Assets\Storage\DBFile)` field (see above) which stores the underlying asset content.																		|
-| `ShowInSearch` | Whether the file should be shown in search results, defaults to '1'. See ["Tutorial 4 - Site Search"](/tutorials/site_search) for enabling search. |
-| `ParentID`     | The ID of the parent Folder that this File/Folder is in. A ParentID of '0' indicates that this is a top level record.								|
-| `OwnerID`      | The ID of the Member that 'owns' the File/Folder (not related to filesystem permissions).                                                            |
+File versioning is extended with the [silverstripe/versioned](https://github.com/silverstripe/silverstripe-versioned/)
+module, which provides not only a separate draft and live stages for any file, but also allows a complete file
+history of modifications to be tracked.
 
+To support this feature the [api:SilverStripe\Assets\AssetControlExtension] provides support for tracking
+references to physical files, ensuring published assets are accessible, protecting non-published assets,
+and archiving / deleting assets after the final reference has been deleted.
 
-## Management through the "Files" section of the CMS
+### Configuring file ownership
 
-If you have the CMS module installed, you can manage files, folders and images in the "Files" section of the CMS. Inside this section, you will see a list of files and folders like below:
+When working with files attached to other versioned dataobjects it is necessary to configure ownership
+of these assets from the parent record. This ensures that whenever a Page (or other record type)
+is published, all assets that are used by this record are published with it.
 
-![](../../_images/assets.png)
+For example:
 
-You can click on any file to edit it, or click on any folder to open it. To delete a file or a folder, simply click the red 'X' symbol next to it. If you click to open a folder, you can go back up one level by clicking the 'up' arrow above the folder name (highlighted below):
+```php
+<?php
+
+use SilverStripe\Assets\Image;
+use SilverStripe\CMS\Model\SiteTree;
+
+class Page extends SiteTree
+{
+    private static $has_one = [
+        'Banner' => Image::class,
+    ];
+    private static $owns = ['Banner'];
+}
+```
 
-![](../../_images/assets_up.png)
+### Avoid exclusive relationships
 
-Once you click to edit a file, you will see a form similar to the one below, in which you can edit the file's title, filename, owner, or even change which folder the file is located in:
+Due to the shared nature of assets, it is not recommended to assign any 1-to-many (or exclusive 1-to-1) relationship
+between any objects and a File. E.g. a Page has_many File, or Page belongs_to File.
+These would require either a subclass, or extension on the File dataobject to create the appropriate
```

Comment:
Actually, instead of saying "Don't do this" I'm going to remove the reference to the undesirable behaviour. :) We don't want users to add these kinds of extensions so let's not lead them in this direction.